### DISCUSSION TITLE
안녕하세요 ebook 관련해서 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by ulzcq** [fsoftwareengineer/todo-application#133] [질문과 답변]
안녕하세요 작가님, 알라딘 ebook을 구매해서 보고있는데 깃헙 코드랑 다른 것 같습니다. 
ebook에는 대부분 함수기반으로 자바스크립트가 작성되어있는데 깃헙은 생성자로 작성되어있네요..  
전자책 출간일은 22년 9월 30일로 표기되어있습니다. 깃헙 코드는 따로 업데이트 하신건가요?

![image](https://user-images.githubusercontent.com/43891587/197015276-5fbc4bbf-2991-4189-8019-9a070be8dfad.png)

**Comment by fsoftwareengineer** [comment]
안녕하세요. 혹시 개정판 리파지토리를 확인 해 보셨나요? https://github.com/fsoftwareengineer/todo-application-revision2

**Reply by ulzcq** [reply]
헉 리포지토리가 따로 있는지 몰랐습니다ㅠㅠ 정말 감사합니다!
